Documentation/contributing/testing/e2e.rst

```diff
@@ -211,3 +211,14 @@ has been resolved):
     # make modules_install
 
 Finally, you can use the instructions from :ref:`the previous chapter<test_cilium_on_lvh>` to run and to test Cilium.
+
+Cleaning up tests
+^^^^^^^^^^^^^^^^^
+
+For cleaning up, you can simply delete the namespace for connectivity tests.
+
+.. code-block:: shell-session
+
+    $ kubectl delete ns cilium-test
+
+if test namespace is specified with `--test-namespace`, make sure replace ``cilium-test`` (default).
```

Comment:
```suggestion
If you specified the test namespace with ``--test-namespace``, make sure to replace ``cilium-test`` (default).
```
A style suggestion.
Also, the CI is failing (https://github.com/cilium/cilium/actions/runs/7646311701/job/20834853707?pr=30428) because of single backticks here, they should be double.

Reply:
addressed in https://github.com/cilium/cilium/pull/30428/commits/05a8e9c10641b39a15be8bb20a1aad95873d7a4f